helm-charts/nxrm-ha/values.yaml

```diff
@@ -0,0 +1,104 @@
+# DO NOT MODIFY. This is produced by template.
+---
+statefulset:
+  name: nexusrepo-statefulset
+  clusterName: nexusrepo-cluster
+  replicaCount: 2
+  container:
+    imageName: registry.connect.redhat.com/sonatype/nexus-repository-manager:3.58.1-ubi-1
+    resources:
+      # See help documentation, these are minimum system requirements
+      requests:
+        cpu: 4
+        memory: "8Gi"
+      limits:
+        cpu: 4
+        memory: "8Gi"
+    containerPort: 8081
+    pullPolicy: IfNotPresent
+    terminationGracePeriod: 20
+    env:
+      clustered: true
+      install4jAddVmParams: "-Xms2703m -Xmx2703m"
+      user: nexus
+      password: nexus
+      jdbcUrl: jdbc:postgresql://your_database_server:5432/nexus
+      nexusInitialPassword: nexusabc
+  livenessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 30
+    path: /
+    periodSeconds: 30
+  readinessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 30
+    path: /
+    periodSeconds: 30
+  imagePullSecret: {}
+ingress:
+  name: nexus-ingress
+  enabled: "false" #intentionally a string because it's used to configure operator via env variable
+  host: "example.com"
+  defaultRule: "false"
+  additionalRules: null
+  tls:
+    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    secretName: "tlsSecretName"
+  dockerSubdomain: false
+  annotations: null
+  dockerIngress:  #Ingress for Docker Connector - comment out if you don't use docker repositories
+    name: nexus-docker-ingress
+    host: "example.com"
+    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    annotations: null
+nexusData:
+  pv:
+    directoryPrefix: /var/nexus-repo-mgr-work-dir/work
+  pvc:
+    size: 100Gi
+    accessMode: ReadWriteOnce
+  storageClass:
+    name: nexusrepo-storage
+    reclaimPolicy: Retain
+  local:
+    enabled: "true" #intentionally a string because it's used to configure operator via env variable
+    accessMode: ReadWriteOnce
+    storageClass:
+      provisioner: kubernetes.io/no-provisioner
+    pv:
+      name: nexusrepo-local-storage-pv
+      size: 200Gi
+      matchNodes: "example-node-1 example-node-2 example-node-3"
+  nfs:
+    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    accessMode: ReadWriteMany
+    storageClass:
+      provisioner: nfs-provisioner
+    pv:
+      name: nexusrepo-nfs-storage-pv
+      size: 200Gi
+      mountPath: /nexusdata/mountPath #Path where blobs should be stored on your NFS drive
```

Comment:
Should the default location really be `/opt/sonatype-work/nexus3/blobs/` or similar?

Reply:
Makes sense, will update

Reply:
This was actually meant to be the exported paths on the server: https://github.com/sonatype/operator-nexus-repository/blob/80309a0497b8d80f904f246ce43ea40f5ee1bd4c/helm-charts/nxrm-ha/values.yaml